<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DivineTextEditor</class>
 <widget class="QMainWindow" name="DivineTextEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>919</width>
    <height>761</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Text Editor</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>3</x>
      <y>30</y>
      <width>911</width>
      <height>651</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>690</y>
      <width>133</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Editor Color To Black</string>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>191</width>
      <height>28</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="exit_button">
       <property name="styleSheet">
        <string notr="true">QPushButton{
background-color : blue;
color : white;
border-radius : 2px;
}</string>
       </property>
       <property name="text">
        <string>Exit</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="open_button">
       <property name="styleSheet">
        <string notr="true">QPushButton{
background-color : blue;
color : white;
border-radius : 2px;
}</string>
       </property>
       <property name="text">
        <string>Open</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="newfile_button">
         <property name="styleSheet">
          <string notr="true">QPushButton{
background-color : blue;
color : white;
border-radius : 2px;
}</string>
         </property>
         <property name="text">
          <string>New File</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="save_button">
         <property name="styleSheet">
          <string notr="true">QPushButton{
background-color : blue;
color : white;
border-radius: 2px;
}</string>
         </property>
         <property name="text">
          <string>Save</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>919</width>
     <height>26</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">

</string>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_As"/>
    <addaction name="actionExit"/>
    <addaction name="actionPrint"/>
    <addaction name="actionNew"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionFind"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionZoom_out"/>
    <addaction name="actionZoom_in"/>
   </widget>
   <widget class="QMenu" name="menuAdvanced">
    <property name="title">
     <string>Advanced</string>
    </property>
    <addaction name="actionAuto_Save"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuView"/>
   <addaction name="menuAdvanced"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="text">
    <string>Save As</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionPrint">
   <property name="text">
    <string>Print</string>
   </property>
  </action>
  <action name="actionFind">
   <property name="text">
    <string>Find</string>
   </property>
  </action>
  <action name="actionZoom_out">
   <property name="text">
    <string>Zoom out</string>
   </property>
  </action>
  <action name="actionZoom_in">
   <property name="text">
    <string>Zoom in</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionAuto_Save">
   <property name="text">
    <string>Auto Save</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
